scanner/Makefile

```diff
@@ -269,7 +269,7 @@ certs-namespace := $(SCANNER_CERTS_NAMESPACE)
 certs-t := $(foreach name,$(certs-names),$(certs-d)/$(name)/.verified)
 
 # Subject prefix.
-certs-subj := /C=US/ST=North Carolina/L=Raleigh/O=Red Hat, Inc./OU=Scanner
+certs-subj := /C=US/ST=North Carolina/L=Raleigh/O=Red Hat, Inc./OU=SCANNER_V4_SERVICE
```

Comment:
Instead of creating this new 'testing only' `SCANNER_V4_SERVICE` identity, would it make more sense instead to generate multiple certs aligned to the current identities (`SCANNER_V4_INDEXER_SERVICE` and `SCANNER_V4_MATCHER_SERVICE`) and use the appropriate one as needed?

Outside of testing don't believe we have plans to run in combo mode right?

Or if keeping this perhaps `SCANNER_V4_COMBO_SERVICE` makes more sense.

Reply:
I'll rename to COMBO